language/move-core/types/src/u256.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fmt;
+
+use primitive_types::U256;
```

Comment:
Have we finalized this choice?

Reply:
No. The plan was to use something for now, wrap it in a new struct, and impl the necessary traits.
The underlying struct can then be swapped out as needed.